Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/MaterialPropertyUtil.cpp

```diff
@@ -222,31 +222,15 @@ namespace AtomToolsFramework
         return true;
     }
 
-    AZStd::string GetExteralReferencePath(const AZStd::string& exportPath, const AZStd::string& referencePath, const uint32_t maxPathDepth)
+    AZStd::string GetExteralReferencePath(
```

Comment:
suggestion: This might be a bigger change not have to be part of this PR but here we could actually return an `AZ::Path` instead of a `string`?